dotcom-rendering/scripts/webpack/browser-targets.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * This is a helper file for SWC to
+ * 1) derive browser targets from @guardian/browserslist-config
+ * 2) upgrade any unsupported browsers as described below.
+ */
+const path = require('path');
+const browserslist = require('browserslist');
+const getTargetsFromBrowsersList =
+	require('@babel/helper-compilation-targets').default;
+
```

Comment:
It would be lovely if this file was a module, and type-checked:

```js
// @ts-check
import path from 'node:path';
import browserlist from 'browserlist';
import getTargetsFromBrowsersList from '@babel/helper-compilation-targets';
```

Reply:
Good idea thanks I'll do it as a follow up